base_comment_template/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -1,28 +1,22 @@
 # Copyright 2020 NextERP Romania SRL
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 from openupgradelib import openupgrade
 
 
 @openupgrade.migrate()
 def migrate(env, version):
     # Not tested
-    properties = env["ir.property"].search(
-        [
-            (
-                "fields_id",
-                "=",
-                env.ref("base.field_res_partner_property_comment_template_id").id,
-            )
-        ]
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO base_comment_template_res_partner_rel
+        (res_partner_id, base_comment_template_id)
+        SELECT SPLIT_PART(ip.res_id, ',', 2) AS res_partner_id,
+        SPLIT_PART(ip.value_reference, ',', 2) AS base_comment_template_id
```

Comment:
I think you have to do a cast of this to integer:
```suggestion
        SPLIT_PART(ip.value_reference, ',', 2)::int AS base_comment_template_id
```